hw2/bor/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using bor;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Tree tree = new Tree();
```

Comment:
```suggestion
        var tree = new Tree();
```
Не надо писать один и тот же тип дважды